network/web3bridge.js

```diff
@@ -40,11 +41,9 @@ class Web3Bridge {
     constructor(ctx) {
         this.ctx = ctx;
         this.log = ctx.log.child({module: 'Web3Bridge'});
-        this.connectionString = this.ctx.config.network.web3;
-        this.network_id = this.ctx.config.network.web3_network_id;
-        this.chain_id = this.ctx.config.network.web3_chain_id;
+        this.connectionString = config.get('network.web3');
         this.address = this.ctx.wallet.getNetworkAccount();
-        this.web3_call_retry_limit = this.ctx.config.network.web3_call_retry_limit || 4;
+        this.web3_call_retry_limit = config.get('network.web3_call_retry_limit') || 4;
```

Comment:
can we use a constant here?

```
// top of file
const DEFAULT_WEB3_RETRY_LIMIT =4 

// ...
 
this.web3_call_retry_limit = config.get('network.web3_call_retry_limit') || DEFAULT_WEB3_RETRY_LIMIT;
```

Reply:
We no longer need the default value here. It is already specified in the `config/default.yaml` file.